code/modules/power/tesla/energy_ball.dm

```diff
@@ -14,6 +14,38 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 										/obj/structure/particle_accelerator/end_cap,
 										/obj/machinery/field/containment,
 										/obj/structure/disposalpipe,
+										/obj/structure/sign,
```

Comment:
Pretty sure this list works for all children, I mean just look at the simple animal entry.